prereqs

```diff
@@ -2,6 +2,19 @@
 
 PREREQS=$(basename "$0")
 
+has_awk() {
```

Comment:
Why not use has() instead of a new function for awk? I did that for curl in the update function, it seemed to be ok.